.circleci/config.yml

```diff
@@ -218,7 +218,8 @@ jobs:
                 command: |
                   docker login -u="${QUAYIO_USERNAME}" -p="${QUAYIO_PASSWORD}" quay.io
                   docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
-                  docker push quay.io/hmpps/<<parameters.app_name>>:$APP_VERSION
+                  docker tag hmpps/<<parameters.app_name>>:$APP_VERSION quay.io/hmpps/<<parameters.app_name>>:latest
+                  docker push --all-tags quay.io/hmpps/<<parameters.app_name>>
```

Comment:
maybe you have to do `docker image push --all-tags quay.io/hmpps/<<parameters.app_name>>` as i can see the tests erroring on the flag --all-tags

Reply:
It's building using circle:python:3 which I assume takes the latest 3.x.x, and 20.10.0 came in -

https://circleci.com/developer/images/image/cimg/python:3.8.9

So I'd expect it to be in...

Reply:
Server Engine Details:
  Version:          24.0.9

Odd!

Reply:
Maybe try `docker image push ...`